src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -154,11 +157,17 @@ const constructPageFetchUrl = ({
 
   if (isLocal) {
     switch (pageType) {
-      case ARTICLE_PAGE:
+      case ARTICLE_PAGE: {
+        const isCpsId = isCpsIdCheck(id);
+        if (isCpsId) {
+          fetchUrl = Url(`/${id}`);
+          break;
+        }
         fetchUrl = Url(
           `/${service}/articles/${id}${variant ? `/${variant}` : ''}`,
         );
         break;
+      }
```

Comment:
Shall we shorten the inference and assignment as below for brevity?
This is opinionated so I'm happy to keep the code block as is.
```suggestion
      case ARTICLE_PAGE:
        fetchUrl = Url(
          isCpsIdCheck(id)
            ? `/${id}`
            : `/${service}/articles/${id}${variant ? `/${variant}` : ''}`,
        );
        break;
```

Reply:
Good spot!